packages/css/src/grid/grid-cell.scss

```diff
@@ -5,14 +5,46 @@
 
 @import "../../utils/breakpoint";
 
-@media screen and (width > $amsterdam-breakpoint) {
-  @for $i from 1 through 12 {
-    .amsterdam-grid-column-start-#{$i} {
+.amsterdam-page-grid > .amsterdam-grid-cell-full-width {
+  grid-column: 1 / -1;
+}
+
+@for $i from 1 through 12 {
+  .amsterdam-page-grid > .amsterdam-grid-span-#{$i} {
+    grid-column-end: span $i;
+  }
+
+  .amsterdam-page-grid > .amsterdam-grid-start-#{$i} {
+    grid-column-start: $i;
+  }
+
+  @media screen and (width < $amsterdam-breakpoint-medium) {
```

Comment:
The [range syntax](https://caniuse.com/css-media-range-syntax) support is like 86%. Maybe we can use min-width just to be save?